### ISSUE TITLE
More results

### ISSUE THREAD

**Issue by xialeiliu** [wuyuebupt/LargeScaleIncrementalLearning#1]
1. Could you provide the results with fixed number of exemplars per class (e.g. 20)?
2. Could you provide the top-1 accuracy as well? I would like to use it as reference.

Thanks in advance.

**Comment by wuyuebupt** [comment]
@xialeiliu 

1. The results are with fixed number of exemplars per class (20). 

2. We don't evaluate the top-1 accuracy on ImageNet dataset. But I leave the code there. You should be able to get the top-1 results by changing following lines in the file `resnet.py`.

line 496 to 500
```
      ### top-1 accuracy
      # accuracy = tf.metrics.accuracy(
      #     tf.argmax(label_all_classes, axis=1), predictions['classes'])
      ### top-5 for imagenet 
      accuracy = tf.metrics.mean(tf.nn.in_top_k(predictions=pred_all_classes, targets=tf.argmax(label_all_classes, axis=1), k=5))
```

line 527 to 534
```
      ### top-1 accuracy
      # accuracy = tf.metrics.accuracy(
      #     tf.argmax(label_all_classes, axis=1), predictions['classes'])
      # accuracy_before = tf.metrics.accuracy(
      #     tf.argmax(label_all_classes, axis=1), predictions['classes_before'])
      ### top-5 accuracy
      accuracy = tf.metrics.mean(tf.nn.in_top_k(predictions=pred_all_classes, targets=tf.argmax(label_all_classes, axis=1), k=5))
      accuracy_before = tf.metrics.mean(tf.nn.in_top_k(predictions=pred_all_classes_before, targets=tf.argmax(label_all_classes, axis=1), k=5))

```

Hope this help.

**Comment by xialeiliu** [comment]
Thanks!
